libavutil/hwcontext_vulkan.c

```diff
@@ -1349,7 +1349,7 @@ static int vulkan_device_init(AVHWDeviceContext *ctx)
         }
     }
 
-    err = load_functions(ctx, 1, 0);
```

Comment:
I don't know, author didn't mention in his patch. But without using 1, there will be an segmentation error because vk->CreateCommandPool is not load. 
Segmentation error occur in this line:
https://github.com/FFmpeg/FFmpeg/blob/2fac1e370c9cf48b2e9d4a7f2c0d7236017f9bbe/libavutil/hwcontext_vulkan.c#L1018

Reply:
Without any of you patch, it will not have a crash, right?

Reply:
yes

Reply:
I submit a ticket. https://trac.ffmpeg.org/ticket/9279#ticket.
Temporarily use this patch before they fix the issue.